.github/workflows/publish.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '0.*'
+      - '1.*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      attestations: write
+      contents: read
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+
+      - name: Build dist packages
+        run: uv build
+
+      - name: Attest build provenance
+        uses: actions/attest-build-provenance@v2.2.3
+        with:
+          subject-path: 'dist/*'
```

Comment:
Feel free to drop this if you'd prefer. There are still advantages over the attestation that's built into the PyPI publish action, but they will be going away as the PyPI system gets improvements, so I think it would be reasonable to just wait for those.

I feel like it's more worth it for ops, which is used in production, to have both for now, than Jubilant, where it seems unlikely that you'd verify your testing dependencies, although you probably ought to.